src/main/java/org/jabref/gui/DialogService.java

```diff
@@ -272,4 +273,6 @@ boolean showConfirmationDialogWithOptOutAndWait(String title, String content,
      * @return the selected file or an empty {@link Optional} if no file has been selected
      */
     Optional<Path> showFileOpenFromArchiveDialog(Path archivePath) throws IOException;
+
+    void show(BaseDialog<?> aboutDialogView);
```

Comment:
I would rename it to `showCustomDialog` to be coherent with the above method. Or rename that to `showAndWait`...yeah maybe the later?

Please also rename the argument, and add a bit of documentation.

Reply:
> I would rename it to `showCustomDialog` to be coherent with the above method. Or rename that to `showAndWait`...yeah maybe the later?
> 
> Please also rename the argument, and add a bit of documentation.

Decided to go with `showCustomDialog()` as it seems to be more consistent with other methods in that interface.